shared/rakefiles/vars.rb

```diff
@@ -81,7 +85,7 @@ def self.set_versions()
       ENV['TF_VAR_dataloader_checksum'] = versions['gpii-dataloader'].split('@')[1]
     end
   end
-  def self.create_dir_if_not_exists(path)
+  def self.create_directory_if_not_exists(path)
```

Comment:
Doesn't look like anything uses this method anymore. Can you remove it and its associated tests?